system/settings/rbp.xml

```diff
@@ -72,6 +72,16 @@
       </group>
     </category>
     <category id="audiooutput">
+      <group id="0">
+        <setting id="audiooutput.useanalogamplifier" type="boolean" label="38122" help="38123">
+          <level>0</level>
+          <default>false</default>
+          <control type="toggle" />
+          <updates>
```

Comment:
This change allow changing "use amplifier" option in UI. What exactly do you mean?